testOPHD/MapOffsetTests.cpp

```diff
@@ -0,0 +1,105 @@
+#include "OPHD/Map/MapOffset.h"
```

Comment:
As `MapOffset` is expected to be in library code, and the unit tests for that library are in a separate project, it would be more appropriate for the `#include` to use `<>` rather than `""`.